content/docs/getting-started/event-driven-architectures.md

```diff
@@ -27,7 +27,7 @@ This extra information is often referred to as _event payload_ or _message paylo
 
 ![](/images/diagrams/simple-event-driven.png)
 
-In most cases, Event-Driven Architectures (EDAs) are broker-centric, like in the diagram above. In it you can find some new concepts, so let's go through them now.
+In most cases, Event-Driven Architectures (EDAs) are broker-centric, like in the diagram above. You can find some new concepts in it, so let's go through them now.
```

Comment:
Thanks. Much easier to read, it is.

![](https://media2.giphy.com/media/3ornk3ifPpyCwE8Ti8/giphy.gif?cid=790b7611e249ce71c0e74936d086365affeaaf2e6b3f6ed3&rid=giphy.gif)